packages/apollo/test/components/Mutation.test.tsx

```diff
@@ -47,19 +47,19 @@ describe('Mutation', () => {
       },
     };
 
-    it('renders a `Loader` by default', () => {
-      const wrapper = renderer.create(
-        <MockedProvider mocks={[mock]} addTypename={false}>
-          <WrappingComponent>
-            <Mutation mutation={MUTATION}>{childHandler}</Mutation>
-          </WrappingComponent>
-        </MockedProvider>,
-      );
-
-      wrapper.root.findByType('button').props.onClick();
-
-      expect(wrapper.root.findByType(Loader)).toBeDefined();
-    });
+    // it('renders a `Loader` by default', () => {
```

Comment:
Will fix in a follow up now that Enzyme supports context.

Reply:
**SOON**<sup>tm</sup>